### ISSUE TITLE
[Accessibility]A11y_Java Tooling for IntelliJ/Eclipse_ Azure Sign In_ Keyboard: Keyboard focus is not going to the “ Azure Sign In” button when we close the pop up. Focus loss is observed after closing the pop up.

### ISSUE THREAD

**Issue by Accessibility56** [microsoft/azure-tools-for-java#3331]
[Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier

GitHub Tags: #A11yTCS; #A11yMAS; #DesktopApp; #Win10; #A11y_Java Tooling for IntelliJ/Eclipse-Stay Healthy; #MAS2.4.3; #Keyboard; #Java tooling for IntelliJ/Eclipse; #C+E-H

**Environment Details:**
Download and Install IntelliJ IDEA from https://docs.microsoft.com/en-us/java/azure/intellij/azure-toolkit-for-intellij-create-hello-world-web-app?view=azure-java-stable
IntelliJ IDEA 2019.1.3
Build #IU-191.7479.19

**OS Details:** 
OS: Windows (Version 1903, Build: 18362.175)
**MAS Violated:** MAS2.4.3

**Repro Steps:** 
1) Open IntelliJ IDEA application
2) Press 'Ctrl+ Alt+ S' to open application settings/preferences popup
3) Tab till 'Plugins' tab and press enter
4) Search for 'Azure Toolkit for IntelliJ in the Marketplace' and press enter.
5) Tab till 'Install' button and press enter.
6) Restart IntelliJ IDEA application.
7)Tab till “Azure Sign In” button and press enter.
8)Tab till “Cancel” button and press enter.
9)Verify the focus order after closing the pop up.

**Actual Result:**
Keyboard focus is not going to the “ Azure Sign In” button when we close the pop up. Focus loss is observed after closing the pop up.

**Expected Result:**
Keyboard focus should go the “Azure Sign In” button when we close the popup. There should not be any focus loss.

**User Impact:**
Users find difficulty if focus is not going to the same control after closing the pop up.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={0de7fbe1-ad7e-48e5-bcbb-8d986691e2b9}
[Bug_3331_Keyboard issue.pptx](https://github.com/microsoft/azure-tools-for-java/files/3451240/Bug_3331_Keyboard.issue.pptx)

**Comment by Flanker32** [comment]
This is a common behavior of IntelliJ, icons in tool window will not be focused again when you open a popup and close it, here is a screen shot of IntelliJ component:
![3331](https://user-images.githubusercontent.com/12445236/64228946-01a26f80-cf1b-11e9-8c44-e7dc317f407d.gif)

And @Accessibility56 , could you please share a bit more information about the rule MAS2.4.3, this [link](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc=%7B0de7fbe1-ad7e-48e5-bcbb-8d986691e2b9%7D&cid=6a75a541-4a73-41bf-a819-a6b8bb6b9655) seems mainly focus on the Web app and seems our behavior didn't break the rules.

**Comment by Accessibility56** [comment]
@Flanker32 If we close any pop up focus should stay on the poop up button itself. If this is the common IntelliJ behavior please mark this bug as External to Intellij

**Comment by Flanker32** [comment]
Importance of experience: Low, it's not a common case to reopen the sign window after close it and we hadn't received any customer report about this issue.
Quality of experience: Low, users could open login popup with keyboard through menus, though it may take a bit more time.
Alternate solutions: No alternate technologies or options can be leveraged.
Evidence: All evidence is provided in this issue and external issue.

**Comment by Flanker32** [comment]
Linked to [AAT](https://cela-ra-aat-prod.azurewebsites.net/#/workflow/56452990-0d38-45bc-b475-a62832ce0076), @Accessibility56 please help us review it. thanks!

**Comment by Accessibility56** [comment]
Logged an external bug : https://youtrack.jetbrains.com/issue/IDEA-222107

**Comment by Flanker32** [comment]
Discussed this bug with Ryan and Sarah and approved to close this issue as its external and there is non blocking scenario.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749783
